README.md

```diff
@@ -4,17 +4,17 @@
 For Advent Of Code 1 I choose From 2017 Day 2 <br>
 
 ### Solution
-~~~
+```kotlin
```

Comment:
Code blocks are defined using three backticks (```).

You should include the `kotlin` annotation so that you get syntax highlighting in your code blocks.